### DISCUSSION TITLE
Don't received email from Github Education

### DISCUSSION THREAD

**Discussion by Andriy15** [community/community#52437] [GitHub Education] [answered]
Hello. My name is Andrew. I applied to Github Student Developer Paсk and my application was accepted and I was told that I would receive an email in less than 6 days, but after 6 days I have not received anything

**Comment by samus-aran** [comment, marked as answer]
Hi @Andriy15 

You can check the status of your application [here](https://education.github.com/discount_requests/pack_application). If your application is still pending review then it will appear at the top of the sidebar to the right of the page. If your application has already been approved then you will be redirected to your GitHub Education benefits.

We aim to provide an outcome to your application submission within a reasonable time. This can take up to 3-4 days, however this may be extended during busy periods. We appreciate your patience and understanding.

Hope this helps! Let us know how you get along 👍
